lib/ansible/modules/packaging/os/apt.py

```diff
@@ -50,6 +50,13 @@
     type: int
     default: 12
     version_added: '2.10'
+  update_cache_allow_releaseinfo_change:
+    description:
```

Comment:
remove update_cache from name and add to the description 'this option is only used if update_cache is yes/truye'